recipes/2dfatmic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "2dfatmic" %}
+{% set version = "1.0" %}
```

Comment:
2014.7 ?

Reply:
Nope, the version is clearly 1.0 in the docs & on the website